.vitepress/config.ts

```diff
@@ -8,827 +8,852 @@ const { BASE: base = "/" } = process.env;
+                                        },
+                                        {
+                                            text: "Jailing and slashing mechanics",
+                                            link: "/how-to-guides/celestia-app-slashing",
+                                        },
+                                        {
+                                            text: "Create a Celestia testnet",
+                                            link: "/how-to-guides/instantiate-testnet",
+                                        },
+                                        {
+                                            text: "Helpful CLI commands",
+                                            link: "/how-to-guides/celestia-app-commands",
+                                        },
+                                        {
+                                            text: "Upgrade Monitor",
+                                            link: "/how-to-guides/celestia-app-upgrade-monitor",
+                                        },
+                                        {
+                                            text: "Wallets in celestia-app",
+                                            link: "/how-to-guides/celestia-app-wallet",
+                                        },
+                                        {
+                                            text: "Multisig",
+                                            link: "/how-to-guides/celestia-app-multisig",
+                                        },
+                                        {
+                                            text: "Create a vesting account",
+                                            link: "/how-to-guides/celestia-app-vesting",
+                                        },
+                                    ],
+                                },
+                                { text: "SystemD", link: "/how-to-guides/systemd" },
+                                { text: "Network upgrade process", link: "/how-to-guides/network-upgrade-process" },
+                            ],
+                        },
+                    ],
                 },
                 {
-                  text: "Optimism",
-                  collapsed: true,
-                  items: [
-                    {
-                      text: "Intro to OP Stack integration",
-                      link: "/developers/intro-to-op-stack",
-                    },
-                    {
-                      text: "Run an OP Stack devnet posting Celestia",
-                      link: "/developers/optimism",
-                    },
-                    {
-                      text: "OP Stack testnets",
-                      collapsed: true,
-                      items: [
+                    text: "Build whatever",
+                    link: "/how-to-guides/build-whatever",
```

Comment:
`celestia.org/how-to-guides/build-whatever` is clear and also has a nice ring. thank you @Mackenzie-OO7 and Diataxis!